minpy/nn/solver.py

```diff
@@ -4,35 +4,39 @@
 from __future__ import print_function
 import time
 
+import minpy
+
 from minpy.nn import optim, init
 from minpy import core
 import minpy.numpy as np
 
+from minpy.visualize.writer import LegacySummaryWriter as SummaryWriter
+import minpy.visualize.summaryOps as summaryOps
 
 class Solver(object):
     """
     A Solver encapsulates all the logic necessary for training classification
     models. The Solver performs stochastic gradient descent using different
     update rules defined in optim.py.
-
+    
```

Comment:
>  ************* Module summaryOps
>  
> =====1/1 python files passed check=====
> All passed!

Quite weird. Would you mind waiting until tomorrow? I will come tomorrow afternoon and maybe meeting face to face will be more helpful?